### PR TITLE
Task Queue for Command_Gateway

### PR DESCRIPTION
**Pull request by TimWDev**

### CODE REVIEW

File: actors/command_gateway/interface.did [hunk 1]
```diff
@@ -57,7 +57,13 @@ type StringResult = variant {
 service : {
     auth: (AuthCommand) -> (NullResult);
     set_canister_targets: (text, text, text, text) -> (NullResult);
-    process_event: (TopicPushEnvelope) -> ();
+    process_account_event: (TopicPushEnvelope) -> ();
```
Comment (SuddenlyHazel) [inline actors/command_gateway/interface.did:60]: lets just make sure we double check the types here line up with our rust types!

File: actors/command_gateway/src/main.rs [hunk 2]
```diff
@@ -189,9 +236,137 @@ pub fn send_tokens(egg: CreateSendSagaEgg) -> Result<String, String> {
...
+            info!(
+                "Command Gateway | process_queue was called, but there was an error! {:?}",
+                e
+            );
+        }
+    }
+}
+
+#[update]
+pub fn request_heartbeat(canister_id: Principal, notify_interval: u64) -> () {
```
Comment (SuddenlyHazel) [inline actors/command_gateway/src/main.rs:294]: Are we using heartbeat anywhere? There is a new cron API I think we can use to more efficiently schedule tasks.

**Review by SuddenlyHazel — APPROVED** [review]
Couple comments. But, overall LGTM!

File: actors/command_gateway/interface.did (hunk 1 above)
Comment (TimWDev) [reply actors/command_gateway/interface.did:60]: Checked, it's 💯

File: actors/command_gateway/src/main.rs (hunk 2 above)
Comment (TimWDev) [reply actors/command_gateway/src/main.rs:294]: Using it here in command_gateway, and also in the notifier service

**Pull request merged**